docs/customer-portal/cp-invoice.md

```diff
@@ -1,9 +1,10 @@
 # Invoice
-This section lists each **Invoice** that is generated for your account. The following functions are available:
 
-+ **Vice Invoice**: Click this to display the invoice in a webpage.
-+ **Download as PDF**: Click this to download a copy of the invoice to your computer. 
+This section lists each generated **Invoice** for your account. The following functions are available:
 
-![alt text][cp-invoice] 
++ **Vice Invoice:** Click here to display the invoice on a webpage.
```

Comment:
Vice Invoice, not sure if you meant to add vice, please check

Reply:
Its **View Invoice**: corrected it

please check:

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-invoice/